packages/rollup-contracts/test/bridge/RollupMessagePasser.spec.ts

```diff
@@ -0,0 +1,34 @@
+import '../setup'
+
+/* External Imports */
+import chai = require('chai')
+import {
+  createMockProvider,
+  deployContract,
+  getWallets,
+  solidity,
+} from 'ethereum-waffle'
+
+chai.use(solidity)
```

Comment:
Is it possible to remove this? Also the `chai = require('chai')` bit. I think this should be handled by importing `setup`